fineract-provider/src/main/resources/sql/migrations/core_db/V327__creditbureau_configuration.sql

```diff
@@ -66,7 +66,7 @@ CREATE TABLE `m_creditbureau_loanproduct_mapping` (
   `is_creditcheck_mandatory` tinyint DEFAULT NULL,
   `skip_creditcheck_in_failure` tinyint DEFAULT NULL,
   `stale_period` INT DEFAULT NULL,
-  `is_active` tinyint DEFAULT NULL,
+  `isActive` tinyint DEFAULT NULL,
```

Comment:
I think you should create a new migration script. But as this is develop can we get away with just fixing the script. 
Any thoughts @vorburger

Reply:
@nikpawar89 @rrpawar96 even on the develop branch we should never update existing SQL migration scripts.. think about it, if you have an existing DB, this script  won't re-run (because Flyway thinks tgat 327 already ran) - and then your code would fail.

Reply:
The changes required in the migration script (V327__creditbureau_configuration.sql) have been done in the new migration script (V327_1__creditbureau_integration.sql) in pr #1235.